src/compile/data/bin.ts

```diff
@@ -25,7 +25,7 @@ export namespace bin {
           }
         },
           // if bin is an object, load parameter here!
-          typeof bin === 'boolean' ? {} : bin
+          isObject(bin) ? bin : {}
```

Comment:
why do you need to change this line.  (FYI, below in this file, you use `typeof bin === 'boolean'`)